pkg/virt-config/config_test.go

```diff
@@ -482,4 +482,18 @@ var _ = Describe("ConfigMap", func() {
 		emulation = clusterConfig.IsUseEmulation()
 		Expect(emulation).To(BeFalse())
 	})
+
+	table.DescribeTable("when feature-gate", func(openFeatureGates string, isLiveMigrationEnabled, isSRIOVLiveMigrationEnabled bool) {
+		clusterConfig, _, _, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.FeatureGatesKey: openFeatureGates},
+		})
+
+		Expect(clusterConfig.LiveMigrationEnabled()).To(Equal(isLiveMigrationEnabled))
+		Expect(clusterConfig.SRIOVLiveMigrationEnabled()).To(Equal(isSRIOVLiveMigrationEnabled))
+	},
+		table.Entry("LiveMigration and SRIOVLiveMigration are closed, both should be closed", "", false, false),
+		table.Entry("LiveMigration and SRIOVLiveMigration are open, both should be open", "LiveMigration,SRIOVLiveMigration", true, true),
```

Comment:
Can you use `virtconfig.LiveMigrationGate` and `virtconfig.SRIOVLiveMigrationGate` ?

Reply:
Done